client/my-sites/site-settings/wpcom-site-tools-settings/index.jsx

```diff
@@ -34,6 +36,7 @@ const SiteSettingsGeneral = ( {
 				isSavingSettings={ isSavingSettings }
 			/>
 		) }
+		{ isEnabled( 'layout/dotcom-nav-redesign' ) && <JetpackMonitor /> }
```

Comment:
I think this needs to be hidden from simple sites. Only Jetpack (includes Atomic) should have access to this.

I assumed this was available to all sites but accidentally switched in to the old screen from an atomic site and got this:

![Screenshot 2024-02-16 at 20-33-13 Security Settings ‹ Site Title — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/9433b4f4-45f9-40a1-9a0d-c4502644754c)

Let's fix this now so we don't need to fix anything later when simple sites get classic view access.

Reply:
Thanks. I have added `isJepack` to the condition.